src/main/resources/server.env

```diff
@@ -1,7 +1,8 @@
 WLP_SKIP_MAXPERMSIZE=true
-VCAP_SERVICES={"retrieve_and_rank": [{"name": "retrieve-and-rank-service","label": "retrieve_and_rank","plan": "standard","credentials": {"url": "https://gateway.watsonplatform.net/retrieve-and-rank/api","password": "-RepalceMe-","username": "-ReplaceMe-"}}],"conversation": [{"name": "Conversation-service","label": "conversation","plan": "standard","credentials": {"url": "https://gateway.watsonplatform.net/conversation/api","isStreaming": false,"password": "-ReplaceMe-","username": "-ReplaceMe-"}}]}
+VCAP_SERVICES={"retrieve_and_rank": [{"name": "retrieve-and-rank-service","label": "retrieve_and_rank","plan": "standard","credentials": {"url": "https://gateway.watsonplatform.net/retrieve-and-rank/api","password": "-RepalceMe-","username": "-ReplaceMe-"}}],"conversation": [{"name": "Conversation-service","label": "conversation","plan": "standard","credentials": {"url": "https://gateway.watsonplatform.net/conversation/api","isStreaming": false,"password": "-ReplaceMe-","username": "-ReplaceMe-"}}],"cloudantNoSQLDB": [{"name":"Cloudant NoSQL DB","label":"cloudantNoSQLDB","credentials":{"username":"-ReplaceMe-","password":"-ReplaceMe-","url":"-ReplaceMe"}}]}
```

Comment:
can we take this out? I'm thinking we will only want to enable on bluemix so I'm thinking this might confuse users even more when they have to manually edit in the service creds